### DISCUSSION TITLE
[Question] Is there any documents relared 'Raw Deployment Installation'?

### DISCUSSION THREAD

**Discussion by smkim17** [kubeflow/kfserving#2902] [General]
I installed KServe using Raw Deployment Installation
https://kserve.github.io/website/master/admin/kubernetes_deployment/

I found it doesn't use Istio VirturalService and Gateway. It seems very different from the basic structure of Kserve. Is there any documentation about the overall structure of kserve when installing that method?

**Comment by yuzisun** [comment]
For Raw deployment installation it uses the kubernetes ingress resource with limited routing capacity but in future we might be able to align with gateway API when it is GA.

**Comment by smkim17** [comment]
> For Raw deployment installation it uses the kubernetes ingress resource with limited routing capacity but in future we might be able to align with gateway API when it is GA.

So the only difference is Knative. Is that right?

**Comment by yuzisun** [comment]
> > For Raw deployment installation it uses the kubernetes ingress resource with limited routing capacity but in future we might be able to align with gateway API when it is GA.
> 
> So the only difference is Knative. Is that right?

Yes that’s right, Knative offers serverless auto scaling and revision based rollout which kserve uses to build on top of.
